Libraries/D3D12RaytracingFallback/src/ComputeAABBs.hlsli

```diff
@@ -104,7 +104,7 @@ void main(uint3 DTid : SV_DispatchThreadID)
             break;
         }
```

Comment:
Ugh, GitHub's awful reviewing tools are stopping me from commenting where I want, but in this file on line 66, left and right node indices are being pulled from the hierarchy buffer. But that buffer is presumably uninitialized in the update case. How is that working?